app/javascript/packages/document-capture/components/in-person-prepare-step.tsx

```diff
@@ -63,7 +53,7 @@ function InPersonPrepareStep({ toPreviousStep, value }) {
       {flowPath === 'hybrid' && <FormStepsButton.Continue />}
       {inPersonURL && flowPath === 'standard' && (
         <div className="margin-y-5">
-          <SpinnerButton onClick={onContinue} isBig isWide>
+          <SpinnerButton type="submit" onClick={onContinue} isBig isWide>
             {t('forms.buttons.continue')}
           </SpinnerButton>
```

Comment:
Are we able to simplify this logic and use `FormStepsButton.Continue` for both cases, esp. now that this page will not direct the user outside the flow?

Reply:
Updated @NavaTim